### DISCUSSION TITLE
Performance and --data-freq-chunk + --g-freq-int

### DISCUSSION THREAD

**Discussion by AKHughes1994** [ratt-ru/CubiCal#475] [Q&A]
Hello, 

I know that the "chunks" determine the amount of data loaded into memory at a time, and the "ints" determine calibration solution intervals, but I'm wondering if there is any improvement in the quality of calibration solutions when `--data-freq-chunk` > `--g-freq-int` or `--data-time-chunk` > `--g-time-int`. 

As an example if I set: 
--data-time-chunk = 30 
--g-time-int = 30 

Would this give the exact same calibration solutions as: 
--data-time-chunk = 60 
--g-time-int = 30 

Given that the solution intervals are the same. If I'm interpreting the parameter correctly, the major difference is that the latter scenario is faster because it loads more data at once. The motivation behind this question is I'm trying to calibrate a very large ms file so I am looking for ways to minimize the memory footprint without sacrificing too much calibration fidelity. 

Cheers,
Andrew

**Comment by JSKenyon** [comment]
The example is correct i.e. you should get identical gain solutions in both cases. Each interval is solved entirely independently of its neighbours.

As a random aside, are you aware of [QuartiCal](https://github.com/ratt-ru/QuartiCal)? It has substantially better memory behaviour than CubiCal so it may be more appropriate for large problems (assuming it has the required functionality).

**Reply by AKHughes1994** [reply]
I am just trying to peel a bright source from my field. I'm assuming Quartical has the functionality and am happy to swap over (I know you re moving away from CubiCal onto QuartiCal). By any chance do you have easy access parameter file conducive for peeling?

**Reply by JSKenyon** [reply]
QuartiCal's config is similar to CubiCal's. Perhaps the simplest thing to do would be to share your current CC parset file. I can use that to throw together a QC equivalent for you.

**Reply by AKHughes1994** [reply]
Sorry for the delayed reply and Thanks for the help! My parset can be found attached. 
[DD_bb.txt](https://github.com/ratt-ru/CubiCal/files/11909220/DD_bb.txt)

**Reply by JSKenyon** [reply]
Here is an approximate QC equivalent: [example.tar.gz](https://github.com/ratt-ru/CubiCal/files/11913862/example.tar.gz)

I haven't set up `output.gain_directory` (where QC will write the gains), so feel free to edit that. If you have any questions about parameters, you can check [the documentation](https://quartical.readthedocs.io/en/latest/), or type `goquartical` in your terminal after installing. Let me know if you run into any problems.
